README.md

```diff
@@ -1,23 +1,36 @@
-A Python library for maintaining and managing DANDI metadata schemata. The
-library helps create and validate DANDI schema-compliant metadata for Dandisets
-and assets.
+# DANDI Schema
 
-To use: `pip install dandischema`
+The dandi-schema repository is a Python library for maintaining and managing DANDI metadata schemata. 
 
-Every Dandiset and associated asset has a metadata object that can be retrieved using
-the DANDI API.
+## Installation
 
-This library uses [Pydantic](https://github.com/samuelcolvin/pydantic) to implement
+`pip install dandischema`
+
+## Description
+
+Every `Dandiset` and associated asset has a metadata object that can be retrieved using
+the DANDI API. This library helps create and validate DANDI schema-compliant metadata for `Dandisets`
+and assets. It uses [Pydantic](https://github.com/samuelcolvin/pydantic) to implement
 all the metadata classes. Schemas are generated on schema modifications and placed into
-[this repository](https://github.com/dandi/schema/tree/master/releases).
+[this repository](https://github.com/dandi/schema/tree/master/releases). 
 
-Dandischema generates JSON schema definitions and also an associated `context.json`
+Dandi-schema generates JSON schema definitions and also an associated `context.json`
 file for JSON-LD compliance of the metadata models.
 
+Important files in this repository include:
 - models.py - contains the models and any changes should be made there
```

Comment:
`any changes should be made there` seems like an odd thing to include in the description of the file. This is presumably *not* the exclusive place to make changes in this repository. Not sure what to do about the phrasing in this PR (maybe nothing) but just pointing it out nonetheless.

Reply:
revised